tokens/custom-spectrum/custom-vars.css

```diff
@@ -53,6 +53,8 @@
   --spectrum-cjk-font: var(--spectrum-code-font-family-stack);
 
   /* static white / black background color for docs containers */
-  --spectrum-docs-static-white-background-color: rgb(15, 121, 125);
-  --spectrum-docs-static-black-background-color: rgb(181, 209, 211);
+	--spectrum-docs-static-white-background-color: rgb(15, 121, 125);
```

Comment:
I'm not sure why this indentation is happening on these token files, it's not like that locally :(